### PR TITLE
Fix docs for shape/points layer properties

### PR DESCRIPTION
**Pull request by guiwitz**
# Description
Some of the docs in the How-to on shape and points layers seem to be outdated. For multiple properties (`edge_width` `face_color` etc) it is indicated that there's also a corresponding "plural" property (`edge_widths` `face_colors` etc) which doesn't actually exist (see e.g. https://napari.org/howtos/layers/shapes.html#changing-shape-edge-and-face-colors). From the source code it looks like there are:
- properties like ```face_color``` that contain a list of colors for each shape/point
- properties like ```current_face_color``` that contain a single color that will be used for the next shape/point

I tried to fix all the places where variable definition was wrong. There's one place where I'm not sure if this is a bug or not. The docs specify very precisely that the opacity for the shape layer is adjusted **per shape** when one is selected but this doesn't seem to be true (see [here](https://napari.org/howtos/layers/shapes.html#shapes-layer-opacity) and there's even a note [here](https://napari.org/guides/layers.html#layer-opacity)). The ```layer.opacity``` is a single value and not a list, and there's no ```current_opacity``` variable, so it really does look like opacity applies to all shapes. I'll complete this PR by fixing this opacity issue if nobody knows better.

## Type of change

- [ x] Bug-fix (non-breaking change which fixes an issue)

# References
Discussed [here](https://napari.zulipchat.com/#narrow/stream/212875-general/topic/face_colors/near/285504543)

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Review by andy-sweet — APPROVED** [review]
Thanks for the catch and the fix here @guiwitz !

**Comment by andy-sweet** [comment]
> The docs specify very precisely that the opacity for the shape layer is adjusted **per shape** when one is selected but this doesn't seem to be true (see [here](https://napari.org/howtos/layers/shapes.html#shapes-layer-opacity) and there's even a note [here](https://napari.org/guides/layers.html#layer-opacity)). The `layer.opacity` is a single value and not a list, and there's no `current_opacity` variable, so it really does look like opacity applies to all shapes. I'll complete this PR by fixing this opacity issue if nobody knows better.

Agreed that this also looks like a bug, so if you can update the layer opacity text too, that would be much appreciated! I'm guessing that may have been the case in the past, but as you found both in behavior and in the code, no longer. I guess we can change the opacity of individual shapes by setting the alpha channel of the edge and face colors.

**Comment by Carreau** [comment]
I pushed a fix to the example which update `current_opacity` to just `opacity`.

File: docs/howtos/layers/points.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ your selection.
 
 Each point can have a different size. You can pass a list or 1-dimensional array
 of points through the size keyword argument to initialize the layer with points
-of different sizes. These sizes are then accessible through the `sizes`
+of different sizes. These sizes are then accessible through the `size`
 property. If you pass a single size then all points will get initialized with
 that size. Points can be pseduo-visualized as n-dimensionsal if the
 `n-dimensional` property is set to `True` or the `n-dimensional` checkbox is
```
Comment (alisterburt) [inline docs/howtos/layers/points.md]: this is also out of date, the attribute is now `out_of_slice_display: bool`

**Review by alisterburt — COMMENTED** [review]
Looks good, thanks @guiwitz! I noticed one small thing which could be added too but could easily be a followup PR, this is great!

File: docs/howtos/layers/points.md (hunk 1 above)
Comment (andy-sweet) [inline docs/howtos/layers/points.md]:
```suggestion
that size. Points can be pseudo-visualized as n-dimensional if the
`out_of_slice_display` property is set to `True` or the `out of slice` checkbox is
```

Comment (andy-sweet) [reply docs/howtos/layers/points.md]: Added a suggestion to fix this and get this PR in soon. If there are not other comments/updates in ~24 hours, I'll commit that and will merge this PR.

**Pull request merged**